app/views/ThreadMessagesView/NoDataFound.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+	ImageBackground, StyleSheet, Text, View
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+import { withTheme } from '../../theme';
+import sharedStyles from '../Styles';
+import { themes } from '../../constants/colors';
+
+const styles = StyleSheet.create({
+	container: {
+		flex: 1
+	},
+	image: {
+		width: '100%',
+		height: '100%',
+		position: 'absolute'
+	},
+	text: {
+		position: 'absolute',
+		top: 60,
+		left: 0,
+		right: 0,
+		textAlign: 'center',
+		fontSize: 16,
+		paddingHorizontal: 24,
+		...sharedStyles.textRegular
+	}
+});
+
+const EmptyRoom = ({ theme, text }) => (
```

Comment:
When I reached this part, I had already made several other PRs refactoring things and that was taking too long.
For this component, I've chosen to refactor it as a later PR instead of creating another blocking point.